src/utils/misc.js

```diff
@@ -66,7 +68,7 @@ const runCommand = (command, args, options) => {
       if (code !== 0) {
         reject(new Error(stderr));
       }
-      resolve(stdout);
+      resolve({ stdout, stderr });
```

Comment:
I assume you've checked we're not expecting this to resolve with only `stdout` anywhere else?

Reply:
I was supposed to. Thanks for the reminder! Fixed in a218ba4.